_studio/shared/include/mfx_config.h

```diff
@@ -51,8 +52,8 @@
 #if !defined(LINUX_TARGET_PLATFORM) || defined(LINUX_TARGET_PLATFORM_BDW) || defined(LINUX_TARGET_PLATFORM_CFL) || defined(LINUX_TARGET_PLATFORM_BXT)
     #if !defined(ANDROID)
         // HW decoders are part of library
-        #define MFX_ENABLE_H264_VIDEO_DECODE
-        #define MFX_ENABLE_H265_VIDEO_DECODE
+        #define MFX_ENABLE_VP8_VIDEO_DECODE
```

Comment:
Idea was to move MFX_ENABLE_H264_VIDEO_DECODE to mfxconfig.h.in, not to add some new ENABLE defines for other codecs!

Reply:
There are compilation errors without MFX_ENABLE_VP... These rows are moved from umc_defs.h